test/functionalTest/harnessFunctions.sh

```diff
@@ -90,49 +90,45 @@ function dMsg()
 
 # ------------------------------------------------------------------------------
 #
-# dbInit - 
+# brokerStart CB - 
```

Comment:
Why this file has need changes? Is related with CB_A_PORT, etc. varialbes or is anythying else needed to make the federation test work?